lib/rescue_from_duplicate/active_record/extension.rb

```diff
@@ -62,12 +67,13 @@ def postgresql_exception_columns(exception)
     end
 
     def sqlite3_exception_columns(exception)
-      extract_columns(exception.message[/columns? (.*) (?:is|are) not unique/, 1])
+      extract_columns(exception.message[/columns? (.*) (?:is|are) not unique/, 1]) || 
+      extract_columns(exception.message[/UNIQUE constraint failed: ([^:]*)\:/, 1])
```

Comment:
Look like sqlite changed it's format. I don't know if there was a major update of sqlite or something, nor if we have a way to test both version :/